modules/virt-about-node-labeling-cpu-features.adoc

```diff
@@ -0,0 +1,131 @@
+// Module included in the following assemblies:
+// * virt/node_maintenance/virt-managing-node-labeling-obsolete-cpu-models.adoc
+
+[id="virt-about-node-labeling-cpu-features_{context}"]
+= About node labeling for CPU features
+
+Through the process of iteration, the list of base CPU features in the minimum CPU model are eliminated from the list of labels generated for the node.
+
+For example, an environment might have two supported CPU models: `Penryn` and `Haswell`. If `Penryn` is specified as the CPU model for `minCPU`, each base CPU feature for `Penryn` is evaluated and compared to each CPU feature supported by `Haswell`.
```

Comment:
Should it say something like "each Penryn feature is removed from cpu features reported for Haswell"?

Reply:
(The example has it right, I just wonder if evaluated and compared are the right verbs here)

Reply:
I do think that phrasing makes it a bit clearer. Thanks for your input!